sui-execution/vm-rework/sui-adapter/src/gas_charger.rs

```diff
@@ -207,6 +209,14 @@ pub mod checked {
             self.gas_status.charge_publish_package(size)
         }
 
+        pub fn charge_upgrade_package(&mut self, size: usize) -> Result<(), ExecutionError> {
```

Comment:
Seems unrelated code?

Reply:
Bad rebase from @dariorussi's fix maybe?

Reply:
The `vm_rework` branch was cut before Dario's fix, so when I rebased the changes from `latest` into `vm_rework` it pulled in the changes to `latest` with it. So I think this is fine/to be expected?

Reply:
Happy to not replicate it over, but it feels kinda weird to not just do a full `execution_layer.py rebase` to replicate the changes over.